pandas/_libs/tslib.pxd

```diff
@@ -1,8 +1,8 @@
 from numpy cimport ndarray, int64_t
 
 from tslibs.conversion cimport convert_to_tsobject
+from tslibs.timedeltas cimport convert_to_timedelta64
```

Comment:
these shouldn’t be here
if something wants to import them they should directly cimport from conversion or timedelta and not tslib